### PR TITLE
Add Ionicons for mobile phone and phone/email feedback (for contact icons)

### PR DESCRIPTION
**Pull request by igor-p**
Ticket: [OA-368](https://reonomy.atlassian.net/browse/OA-368)

I needed to update 2 icons in the property owner tab contact area. `<IconPhonePortraitSolid>` will appear on the left side of a phone number that's identified as a cellphone. And `<IconStarHalfSolid>` will replace the "thumbs up/down" icon that opens the phone/email feedback popover.

~~I'm marking this as `wip` as I'm not yet able import these icons into the `/example/` directory to ensure that they were added~~

***Edit:** I'm able to see the new icons in `examples/`. When I ran `yarn` in `example/` I must have installed the old `.tgz` package, and reinstalling didn't seem to overwrite it (ultimately getting rid of `example/node_modules/` and `example/yarn.lock` and reinstalling did it).

### CODE REVIEW

File: src/ionicons/solidIcons/IconStarHalfSolid.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <SvgIcon viewBox="0 0 24 24" fill="none" {...props}>
+      <path
+        d="M22.5 8.75H14.4375L12 1.25L9.5625 8.75H1.5L8.0625 13.25L5.53125 20.75L12 16.0625L18.4688 20.75L15.9375 13.25L22.5 8.75Z"
+        fill="none"
+        stroke="currentColor"
+        strokeWidth="1.5"
+        strokeLinejoin="round"
+      />
+      <path d="M12 1.25V16.0625L5.53125 20.75L8.0625 13.25L1.5 8.75H9.5625L12 1.25Z" fill="currentColor" />
+    </SvgIcon>
```
Comment (herrethan) [inline src/ionicons/solidIcons/IconStarHalfSolid.tsx:15]: icons like this should have `stroke="currentColor"` set as a `SvgIcon` attribute instead. (see ArrowIcons.tsx)

File: src/ionicons/solidIcons/IconPhonePortraitSolid.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        fill="currentColor"
+      />
+      <path
+        d="M10.75 0H3.25C2.45435 0 1.69129 0.316071 1.12868 0.87868C0.56607 1.44129 0.25 2.20435 0.25 3V21C0.25 21.7956 0.56607 22.5587 1.12868 23.1213C1.69129 23.6839 2.45435 24 3.25 24H10.75C11.5456 24 12.3087 23.6839 12.8713 23.1213C13.4339 22.5587 13.75 21.7956 13.75 21V3C13.75 2.20435 13.4339 1.44129 12.8713 0.87868C12.3087 0.316071 11.5456 0 10.75 0V0ZM12.25 21C12.25 21.3978 12.092 21.7794 11.8107 22.0607C11.5294 22.342 11.1478 22.5 10.75 22.5H3.25C2.85218 22.5 2.47064 22.342 2.18934 22.0607C1.90804 21.7794 1.75 21.3978 1.75 21V3C1.75 2.60218 1.90804 2.22064 2.18934 1.93934C2.47064 1.65804 2.85218 1.5 3.25 1.5H3.78203C3.85456 1.49974 3.92558 1.52067 3.98638 1.56022C4.04717 1.59978 4.09508 1.65624 4.12422 1.72266C4.24068 1.99075 4.43294 2.21898 4.67735 2.37929C4.92177 2.53959 5.2077 2.625 5.5 2.625H8.5C8.7923 2.625 9.07823 2.53959 9.32265 2.37929C9.56706 2.21898 9.75932 1.99075 9.87578 1.72266C9.90492 1.65624 9.95283 1.59978 10.0136 1.56022C10.0744 1.52067 10.1454 1.49974 10.218 1.5H10.75C11.1478 1.5 11.5294 1.65804 11.8107 1.93934C12.092 2.22064 12.25 2.60218 12.25 3V21Z"
+        fill="currentColor"
```
Comment (herrethan) [inline src/ionicons/solidIcons/IconPhonePortraitSolid.tsx:13]: shouldn't need `fill="currentColor"` on these paths

File: src/ionicons/solidIcons/IconPhonePortraitSolid.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@material-ui/core/SvgIcon';
+
+export function IconPhonePortraitSolid(props: SvgIconProps) {
+  return (
+    <SvgIcon viewBox="0 0 14 24" fill="none" {...props}>
```
Comment (herrethan) [inline src/ionicons/solidIcons/IconPhonePortraitSolid.tsx:6]: view box should _always_ be `0 0 24 24`

**Review by herrethan — CHANGES_REQUESTED** [review]
Can we also add the Outline versions of both these icons?

File: .gitignore [hunk 4]
```diff
@@ -41,4 +41,8 @@ typings/
 .DS_Store
 
 # Example react app
-example/yarn.lock
\ No newline at end of file
+example/yarn.lock
+
+# Intellij Artifacts
+.idea/
+*.iml
```
Comment (herrethan) [inline .gitignore:48]: nit: newline

File: src/ionicons/solidIcons/IconPhonePortraitSolid.tsx (hunk 3 above)
Comment (igor-p) [reply src/ionicons/solidIcons/IconPhonePortraitSolid.tsx]: Fixed in a4772c82926b9e8dbaded63a4be20c5305957586

**Comment by igor-p** [comment]
> Can we also add the Outline versions of both these icons?

I added the outline version of `PhonePortrait`, but the star icon is the same in both solid and outline variants.

**Comment by herrethan** [comment]
> I added the outline version of `PhonePortrait`, but the star icon is the same in both solid and outline variants.
That's fine, I believe for the most part the same solid/outline icons have been duplicated thus far, just for consistency. But probably unnecessary.

**Review by herrethan — APPROVED** [review]

**Comment by igor-p** [comment]
> > I added the outline version of `PhonePortrait`, but the star icon is the same in both solid and outline variants.
> 
> That's fine, I believe for the most part the same solid/outline icons have been duplicated thus far, just for consistency. But probably unnecessary.

Ah, that makes sense. Probably good to throw in the alternate version in any case

**Review by herrethan — APPROVED** [review]

**Pull request merged**